ui/src/components/VolumeListTable.js

```diff
@@ -256,6 +256,26 @@ function Table({ columns, data, nodeName, rowClicked, volumeName, theme }) {
           })}
         </thead>
         <Body {...getTableBodyProps()}>
+          {data.length === 0 ? (
+            <HeadRow
+              style={{
+                width: '100%',
+                paddingTop: padding.base,
+                height: '60px',
+              }}
+            >
+              <td
+                style={{
+                  textAlign: 'center',
+                  background: theme.brand.primaryDark2,
```

Comment:
Do we want to use the `primaryDark2` or to use the same background color as the Table (`primary`)

Reply:
Talk with Valamir, since no volume in the node is not an error state. I removed the Exclamation icon and chang the background color to primaryDark1 which the color of table.